src/app/components/feed/feeds.factory.js

```diff
@@ -16,19 +20,25 @@
   /**
    * Factory representing a janus feed
    * @constructor
+   * @memberof module:janusHangouts
    */
   function feedFactory($timeout, DataChannelService, SpeakObserver) {
     return function(attrs) {
       attrs = attrs || {};
       var that = this;
 
+      /** @var {integer} id of feed */
       this.id = attrs.id || 0;
+      /** @var {string} name of user streaming feed */
       this.display = attrs.display || null;
-      this.connection = attrs.connection || null;
+      /** @var {boolean} flag if feed is publishing one, so one that is send from this pc to others */
       this.isPublisher = attrs.isPublisher || false;
+      /** @var {boolean} flag if feed is local screen sharing feed */
       this.isLocalScreen = attrs.isLocalScreen || false;
+      /** @var {boolean} flag if feed is ignored */
       this.isIgnored = attrs.ignored || false;
 
+      var connection = attrs.connection || null;
```

Comment:
ah, doing it next time it is used for main feed once, probably appear after merge. so revert